### ISSUE TITLE
Local game crashes with libgcc error win32 build

### ISSUE THREAD

**Issue by MJLHThomassen** [solarisstar/rpgxEF#78]
**Operating System:** Windows 7
**RPG-X Version:** 2.4 Beta
**Short Description of bug:** 
When you try to load up the Station Iowa map it crashes with this error: http://imgur.com/Sx3JOaQ

**Steps to Reproduce Bug:**
Open up RPG-X with the rpgxEF.x86.exe program and then enter console and type /map station_iowa

UPDATE: I can't get 1 single map to work. I tried rpgxEFded.x86.exe but it just says that the program stopped working.[/size]
UPDATE 2: I can connect to the alpha server successfully but whenever I try to load a map locally it fails.

**Comment by erixom** [comment]
(This is Jack) I tried re downloading the latest build just now but it didn't work.

**Comment by MJLHThomassen** [comment]
Yes, i had to revert the change Telex made because it didn't work (the build didn't run with the changed build flag), so the version currently online is still the old one.

I currently don't have the time to figure out what's wrong here, but I've been having some troubles running localhost servers myself (though a different error then you are getting).I'll see if i can figure this one out during the week.

**Comment by erixom** [comment]
Any Luck?

**Comment by MJLHThomassen** [comment]
Sorry no, I've been swamped with work lately so havn't had much time or patience to take a look at this.

**Comment by erixom** [comment]
Alright

**Comment by erixom** [comment]
Good Luck with you work
